examples/text-classification/run_pl_glue.py

```diff
@@ -72,7 +76,7 @@ def prepare_data(self):
                 logger.info("Saving features into cached file %s", cached_features_file)
                 torch.save(features, cached_features_file)
 
-    def load_dataset(self, mode, batch_size):
+    def get_dataloader(self, mode, batch_size):
```

Comment:
`get_dataloader` has one more arg - `shuffle`

Reply:
Yes, i think wrapping this load_dataset() into a get_dataloader() is the best way to handle this.